config/routes.rb

```diff
@@ -8,6 +8,6 @@
 
     get "translate", to: "translations#index"
     get "about", to: "static#about"
-    root to: "doctors#search"
+    root to: "doctors#search_root"
```

Comment:
Damn I accidentally pressed cancel! Argh. 

I did this so the button would only appear at root, since it seems confusing to have the find location button on the results page. 

I think this might make sense to keep it this way, since having the css rule visibility:hidden might be messy.

The other option is to actually add the button to the page with jQuery.